src/pages/SearchPage/SearchPageFooter.tsx

```diff
@@ -1,16 +1,24 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {View} from 'react-native';
 import ReferralProgramCTA from '@components/ReferralProgramCTA';
 import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
 
 function SearchPageFooter() {
+    const [shouldShowReferralCTA, setShouldShowReferralCTA] = useState(true);
     const themeStyles = useThemeStyles();
 
     return (
-        <View style={[themeStyles.pb5, themeStyles.flexShrink0]}>
-            <ReferralProgramCTA referralContentType={CONST.REFERRAL_PROGRAM.CONTENT_TYPES.REFER_FRIEND} />
-        </View>
+        <>
+            {shouldShowReferralCTA && (
```

Comment:
I don't think this works, does it? I think there is some confusion with the `SearchPage` component: https://expensify.slack.com/archives/C01GTK53T8Q/p1705959664098999?thread_ts=1701232130.505309&cid=C01GTK53T8Q

Reply:
OK, looks like the old file `SearchPage.js` will be getting removed. Thanks!